DevEdu.API/DevEdu.API/Controllers/CourseController.cs

```diff
@@ -146,6 +146,26 @@ public List<CourseTopicOutputModel> SelectAllTopicsByCourseId(int courseId)
             return _mapper.Map<List<CourseTopicOutputModel>>(list);
             
         }
+        // api/course/{courseId}
+        [HttpPut("{courseId}/update")]
+        [Description("updates topics in the course")]
+        [ProducesResponseType(typeof(string), StatusCodes.Status200OK)]
+        public string UpdateCourseTopicsByCourseId(int courseId,  [FromBody] List<CourseTopicUpdateInputModel> topics)
+        {
+            var list = _mapper.Map<List<CourseTopicDto>>(topics);
+            _courseService.UpdateCourseTopicsByCourseId(courseId, list);
+            return "updated";
+
+        }
+        // api/course/{courseId}/topics-delete
+        [HttpDelete("{courseId}/topics-delete")]
```

Comment:
do we really need to delete all? 😱😱😱

Reply:
why not? Maybe for someone will be easier to delete all topics than delete one by one. If you say to delete then I'll delete.